pyproject.toml

```
@@ -1,3 +1,47 @@
+[project]
+name = "readme_renderer"
+version = "41.0"
+description = "readme_renderer is a library for rendering readme descriptions for Warehouse"
+authors = [
+  {name = "The Python Packaging Authority", email = "admin@mail.pypi.org"}
+]
+readme = "README.rst"
+license = {file = "LICENSE"}
+dependencies = ["bleach>=2.1.0", "docutils>=0.13.1", "Pygments>=2.5.1"]
+classifiers = [
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: Apache Software License",
+  "Natural Language :: English",
+  "Operating System :: MacOS :: MacOS X",
+  "Operating System :: POSIX",
+  "Operating System :: POSIX :: BSD",
+  "Operating System :: POSIX :: Linux",
+  "Operating System :: Microsoft :: Windows",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+  "Typing :: Typed"
+]
+requires-python = ">=3.8"
+
+[project.optional-dependencies]
+md = ["cmarkgfm>=0.8.0"]
+
+[tool.setuptools]
+include-package-data = true
+
+[tool.setuptools.packages.find]
+exclude = ["tests", "tests.*"]
+
+[project.urls]
+urls = "https://github.com/pypa/readme_renderer"
```

```suggestion
Home-page = "https://github.com/pypa/readme_renderer"
```
Per https://packaging.python.org/en/latest/specifications/declaring-project-metadata/#urls